File: 백준/Bronze/5086. 배수와 약수/배수와 약수.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);

        while(true){

            int first = scan.nextInt();
            int second = scan.nextInt();

            if(first == 0 && second == 0)
                break;

            if(first % second == 0)
                System.out.println("multiple");

            else if(second % first == 0)
                System.out.println("factor");

            else
                System.out.println("neither");

        }

    }
}

File: 백준/Bronze/5086. 배수와 약수/배수와 약수.cc
#include <bits/stdc++.h>
using namespace std;

int main(){
    ios::sync_with_stdio(0); cin.tie(0);
    
    int A, B;
    
    while(1){
        cin >> A >> B;
        if(A == 0 && B == 0){break;}
        else if((A % B) == 0){cout << "multiple";}
        else if(B % A == 0){cout << "factor";}
        else {cout << "neither";}
        cout << '\n';
    }
    
    return 0;
}